### DISCUSSION TITLE
Deal with search when JS is off

### DISCUSSION THREAD

**Discussion by LucyGwilliamAdmin** [open-sdg/open-sdg#1611] [Ideas]
When JS if off, searching for any terms results in continuous loading wheel.

**To discuss: can we/should we display an error message instead in this instance? or should we completely hide the search?**

Search results when JS is off:
![image](https://user-images.githubusercontent.com/43404358/168619676-8d184f29-76ac-4956-bba1-47e4674453d8.png)

**Comment by LucyGwilliamAdmin** [comment]
@phillipgardiner @otis-bath @brockfanning thoughts please!

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin @phillipgardiner @otis-bath Similar to other cases, the search cannot work without javascript. So we should be hiding the search bar entirely when javascript is off.

**Reply by phillipgardiner** [reply]
@LucyGwilliamAdmin @otis-bath @brockfanning I'll refer back to brocks technical knowledge on this one, but if search cannot be provided in anyway via the use of HTML and possibly CSS then I guess it would have to be hidden. However I think we would need to adjust the interface (particularly on mobile) so you are not just left with a big white gap where the search bar would have been.
